src/commands/cooldown.ts

```diff
@@ -27,30 +30,69 @@ export class CooldownCommand extends Command {
             return { dropCooldown, claimCooldown, dailyCooldown}
         })
 
+        const effects: UserEffect[] = await db.getRepository(UserEffect).find({
+            where: {
+                userId: user.id,
+                activeUntil: MoreThan(new Date())
+            }
+        })
+
         const freeClaimCount = await ItemService.getItemCount(user, 'free claim')
 
+        const embedBuilder = 
+            new EmbedBuilder()
+                .setTitle('Cooldowns')
+                .addFields([
+                    {
+                        name: 'Drop',
+                        value: dropCooldown <= 0 ? 'Ready to use' : `${dropCooldown < 60_000 ? Math.ceil(dropCooldown / 1000) + 'seconds' : Math.ceil(dropCooldown / 60_000) + 'minutes'} remaining`,
+                    },
+                    {
+                        name: 'Claim',
+                        value: (claimCooldown <= 0 ? 'Ready to use' : `${claimCooldown < 60_000 ? Math.ceil(claimCooldown / 1000) + 'seconds' : Math.ceil(claimCooldown / 60_000) + 'minutes'} remaining`) +
+                            (freeClaimCount > 0 ? ` (+${freeClaimCount} free claim${freeClaimCount > 1 ? 's' : ''})` : ''),
+                    },
+                    {
+                        name: 'Daily',
+                        value: dailyCooldown <= 0 
+                        ? 'Ready to use' 
+                        : `${dailyCooldown < (60_000 * 60) ? Math.ceil(dailyCooldown / 60_000) + ' minutes' : Math.ceil(dailyCooldown / (60_000 * 60)) + ' hours'} remaining`,
```

Comment:
probably a good idea to extract a method for formatting the durations as well, same code repeated everywhere